rasa_nlu/train.py

```diff
@@ -42,9 +43,9 @@ def create_argument_parser():
                             "or a directory containing multiple training "
                             "data files.")
 
-    group.add_argument('-u', '--url',
```

Comment:
I'd keep this, but use it as a shortcut to set the endpoint (so keep the `data_endpoint`, but allow the user to overwrite by using this argument). thoughts?

Reply:
yes, I like that idea. so we'd support both ``--endpoints`` with a yaml file and ``--url`` with just a string?

Reply:
yes and the url would just overwrite the endpoints.